jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+   <j-tray-plugin>
+     <v-row>
+       <j-docs-link :link="'https://jdaviz.readthedocs.io/en/latest/dev/ui_plugin_components.html'">
```

Comment:
This is just a placeholder for the document that will be created for this plugin!

Reply:
Not sure if you've rebased on main recently, but the suggested syntax for the plugin headers has changed in #1423 to (see [UI style guide](https://jdaviz.readthedocs.io/en/latest/dev/ui_style_guide.html):

```
<j-tray-plugin
    description='Plugin description.'
    :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#plugin-name'"
    :popout_button="popout_button">
    ...
</j-tray-plugin>
```

Reply:
Made the changes!